atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -245,6 +245,11 @@ boolean allSafeForLogging() {
         return false;
     }
 
+    @Value.Default
+    String timeLockVersion() {
+        return "0.0.0";
+    }
+
```

Comment:
The default should be an identifier that should tell that cluster is not on / moving away from blue greening and leadership should not be snatched because of version diff in this case.